src/components/src/main/java/org/apache/jmeter/visualizers/backend/influxdb/InfluxDBRawBackendListenerClient.java

```diff
@@ -122,8 +122,10 @@ private String createTags(SampleResult sampleResult) {
         // remove surrounding quotes and spaces from sample label
         String label = StringUtils.strip(sampleResult.getSampleLabel(), "\" ");
         String transaction = AbstractInfluxdbMetricsSender.tagToStringValue(label);
+        String threadName = StringUtils.deleteWhitespace(sampleResult.getThreadName());
```

Comment:
This might be time-consuming.
Do you really need a thread name?
Would it be workable if we included a "unique thread id"?

Reply:
Sure, if the thread ID is unique at the Test Plan level, it should work.